### ISSUE TITLE
Doesn't install dependencies after initialization

### ISSUE THREAD

**Issue by silvenon** [yeoman/generator-webapp_DEPRECATED#551]

**Comment by arthurvr** [comment]
Confirmed. Strange. Any ideas?

**Comment by silvenon** [comment]
Nope :cry: I'll get to that at some point, I just wanted to raise an issue so I don't forget.

**Comment by IanLunn** [comment]
Can you elaborate please? I am getting an ENOENT open error. Is that the same issue you are experiencing?

**Comment by silvenon** [comment]
The npm and bower install doesn't initiate, I didn't get an error. What are you getting?

**Comment by IanLunn** [comment]
Not sure if it relates. I've been using webapp successfully for quite some time but today it started throwing errors. I get past `I'm all done. Running bower install & npm install` but when installing jQuery, I get errors such as:

```
ENOTEMPTY ENOTEMPTY, rmdir '/Users/ian/test/bower_components/modernizr/feature-detects'
ENOENT ENOENT, open '/Users/ian/test/bower_components/jquery/.bower.json'
```

I've updated everything to latest version. Checked permissions, ran `npm cache clean` and `bower cache clean` but still the problem persists.
